src/olympia/amo/tests/__init__.py

```diff
@@ -91,6 +91,11 @@
 SQUOTE_ESCAPED = escape("'")
 
 
+# A Storage instance for the filesystem root to be used during tests that read fixtures
+# and/or try to copy them under settings.STORAGE_ROOT.
+root_storage = SafeStorage(location='/')
```

Comment:
Just curious, is there a reason this needs to be set here, as opposed to just doing this at line 469?

Reply:
I think I answered my own question after reading more of the code. Sometimes you want to import this, and sometimes it's available in `TestCase`. I did ask a different, related question below.

Reply:
yep, it's just for the two different use cases.  There's technically nothing stopping us from always importing it, or accessing it as `TestCase.root_storage` outside of a `TestCase` subclass, but the former introduced more unnecessary `import`s and the latter is a little abnormal in our tests (though acceptable if it was, like, one occurrence)